examples/impute/plot_missing_values.py

```diff
@@ -8,6 +8,9 @@
 The median is a more robust estimator for data with high magnitude variables
```

Comment:
rather than just high magnitude, one may also mention explicitly that median is more robust in the presence of outliers